text/0000-clearer-changelogs.md

```diff
@@ -0,0 +1,142 @@
+- Start Date: 2019-02-04
+- RFC PR: (leave this empty)
+- Gatsby Issue: (leave this empty)
+
+# Summary
+
+The aim of this RFC is to make Gatsby's changelogs clearer and more concise, by improving their formatting.
+
+> Note: See the [related RFC on changelog entries](https://github.com/gatsbyjs/rfcs/pull/27)
+
+> Note: this RFC is specifically not about changing Gatsby's release process to semver (or something else). That's a discussion for another RFC.
+
+# Basic Example
+
+## Current changelog example:
+
+---
+
+## [2.0.112](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.111...gatsby@2.0.112) (2019-02-04)
+
+### Bug Fixes
+
+- **gatsby:** update terser-webpack-plugin to avoid webpack error ([#11542](https://github.com/gatsbyjs/gatsby/issues/11542)) ([f945fe8](https://github.com/gatsbyjs/gatsby/commit/f945fe8))
+
+## [2.0.110](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.109...gatsby@2.0.110) (2019-02-01)
+
+**Note:** Version bump only for package gatsby
+
+## [2.0.107](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.106...gatsby@2.0.107) (2019-01-31)
+
+### Bug Fixes
+
+- **gatsby:** always respond with index html without checking path ([#11400](https://github.com/gatsbyjs/gatsby/issues/11400)) ([2f79efe](https://github.com/gatsbyjs/gatsby/commit/2f79efe))
+- **gatsby:** check type when querying by ID ([#11448](https://github.com/gatsbyjs/gatsby/issues/11448)) ([1d95e67](https://github.com/gatsbyjs/gatsby/commit/1d95e67))
+
+### Features
+
+- **gatsby:** Allow specifying sort order per sort field ([#11381](https://github.com/gatsbyjs/gatsby/issues/11381)) ([11e8930](https://github.com/gatsbyjs/gatsby/commit/11e8930))
+
+---
+
+## Proposed changelog example:
+
+---
+
+## [2.0.112](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.111...gatsby@2.0.112) (2019-02-04)
+
+- 🐛 **Bug Fix** - **gatsby:** update terser-webpack-plugin to avoid webpack error [#11542](https://github.com/gatsbyjs/gatsby/issues/11542) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+
+## [2.0.110](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.109...gatsby@2.0.110) (2019-02-01)
+
+- 🧹 **Chore**: update packages peer-dependencies to discard gatsby-alpha [#10252](https://github.com/gatsbyjs/gatsby/pull/10252)
+
+## [2.0.107](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.106...gatsby@2.0.107) (2019-01-31)
+
+- 🐛 **Bug Fix** - **gatsby:** always respond with index html without checking path [#11400](https://github.com/gatsbyjs/gatsby/issues/11400) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+- 🐛 **Bug Fix** - **gatsby:** check type when querying by ID [#11448](https://github.com/gatsbyjs/gatsby/issues/11448) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
```

Comment:
Will changelogs always be one package? Or is it possible that this list would include bug fixes for multiple packages?

If it's multiple packages, I think my preference would be to group by package, then fix type, e.g.:

- **gatsby**
  - 🐛 **Bug Fix** — ...
  - 🚀 **Feature** — ...
- **gatsby-source-graphql**
  - 🧹 **Chore** — ...

If it's single-package-per-changelog, I'm into this.

Reply:
For now they will probably stay 1 changelog per package. The package in this example (and what we use currently) is conventional commit scope (`prefix(scope):`) (but we could discard that if scope is the same as package name - at least I think it's doable)

Reply:
you can create a changelog formatter yourself where you can omit prefix and scope when creating it so we're good.